Blockzilla/PhotoManager.swift

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+protocol PhotoManagerDelegate: class {
+    func photoManager(_ photoManager: PhotoManager, didFinishSavingWithError error: Error?)
+}
+
+class PhotoManager: NSObject {
+    weak var delegate: PhotoManagerDelegate?
+
+    func save(image: UIImage) {
```

Comment:
yep! We'll probably want to implement the delegate in case they've previously denied permission (or not show the save menu at all, but that could be weird). Either way that state would require some UX